### PR TITLE
Updated calculate_proportion.R

### PR DESCRIPTION
**Pull request by Arnaud-Gruss**
I updated the function calculate_proportion so that it takes into account the object Expansion_cz to calculate proportions and associated standard errors.

### CODE REVIEW

**Comment by Arnaud-Gruss** [comment]
Well, it is a real mistery to me, but there are differences between the R script displayed on GitHub: https://github.com/James-Thorson-NOAA/FishStatsUtils/blob/development/R/calculate_proportion.R and the R script that I get when forking and cloning your FishStatsUtils development branch. 
I suspect that there is some kind of lag and that I should be able to fork and clone the latest version of your FishStatsUtils development branch within a few hours. 
The R script here looks good: https://github.com/James-Thorson-NOAA/FishStatsUtils/blob/development/R/calculate_proportion.R , i.e., the five additional lines of code I suggested are present and the function calculate_proportion now takes an additional argument, namely Expansion_cz=NULL.
Long story short, I think we are all good, so I am gonna close the pull request and let me know if there are any additional issues.
